tarpc/Cargo.toml

```diff
@@ -33,9 +33,9 @@ humantime = "2.0"
 log = "0.4"
 pin-project = "1.0"
 rand = "0.7"
-serde = { optional = true, version = "1.0", features = ["derive"] }
+serde = { version = "1.0", features = ["derive"] }
 static_assertions = "1.1.0"
-tarpc-plugins = { path = "../plugins", version = "0.8" }
+tarpc-plugins = { path = "../plugins", version = "0.8", features = ["serde1"] }
```

Comment:
I don't think we want to unilaterally enable the serde1 feature. Otherwise, what's the point in having the feature? :) making it optional can decrease compilation time for those users who do not use serde.